src/components/Handbook/Content/styled.ts

```diff
@@ -11,12 +11,18 @@ import { liBaseStyle } from '../../UI/List/styled'
 export const Renderer = styled.div`
   position: relative;
 
-  & > * {
-    max-width: 38.5rem;
-    width: calc(100% - 16rem);
+  & > *:not(aside) {
+    /* Due to conflicting CSS, Side Notes were having width issues.
+    To prevent it, we must use :not(aside)
+    Aside refer to Handbook Sidenotes  */
```

Comment:
Please add a reference to where we use the aside (CustomComponentParser/styled.ts)